crates/compiler/test_syntax/tests/test_fmt.rs

```diff
@@ -5317,18 +5317,63 @@ mod test_fmt {
     }
 
     #[test]
-    fn multi_line_binary_op_1() {
+    fn multi_line_binary_op_and() {
+        expr_formats_to(
+            indoc!(
+                r"
+                is_last
+                && is_empty
+                && is_loaded
+                "
+            ),
+            indoc!(
+                r"
+                is_last
+                and is_empty
+                and is_loaded
+                "
+            ),
+        );
+
+        expr_formats_same(indoc!(
+            r"
+            is_last
+            and is_empty
+            and is_loaded
+            "
+        ));
+    }
+
+    #[test]
+    fn multi_line_binary_op_or() {
+        expr_formats_to(
+            indoc!(
+                r"
+                is_last
+                || is_empty
+                || is_loaded
+                "
+            ),
+            indoc!(
+                r"
+                is_last
+                or is_empty
+                or is_loaded
+                "
+            ),
+        );
+
         expr_formats_same(indoc!(
             r"
-            isLast
-            && isEmpty
-            && isLoaded
+            is_last
+            or is_empty
+            or is_loaded
             "
         ));
     }
 
     #[test]
-    fn multi_line_binary_op_2() {
+    fn multi_line_binary_op_symbol() {
```

Comment:
I'd really like to see how this interacts parsing wise with `try`, `crash`, and `dbg`